pgbouncer/templates/_pgbouncer.ini.tpl

```diff
@@ -62,13 +62,13 @@ stats_users = {{ $users }}, stats, root, monitor
 
 ;;; Pooler personality
 
-pool_mode = {{ .Values.poolMode }}
-server_reset_query = DISCARD ALL
-;server_reset_query_always = 0
-;ignore_startup_parameters = extra_float_digits
-;server_check_query = select 1
-;server_check_delay = 30
-;application_name_add_host = 0
+pool_mode = {{ .Values.poolerPersonality.poolMode | default "session" }}
```

Comment:
in order to preserve backwards compatibility for chart users that may be doing custom overrides (through a `-f overrides.yaml` file or by pulling in pgbouncer as a requirements chart), i recommend changing this to use `.Values.poolMode` if it is in the dictionary.  could you change this line to
```
pool_mode = {{ .Values.poolMode | default .Values.poolerPersonality.poolMode | default "session" }}
```